corehq/apps/app_manager/views/app_summary.py

```diff
@@ -414,6 +419,23 @@ def get(self, request, domain, app_id):
     def get_case_property_rows(self, case_type):
         return tuple((case_type.name, prop.name, prop.description) for prop in case_type.properties)
 
+    def get_case_type_rows(self, case_type, language):
+        rows = []
+        relationships = ["[{}] {}".format(r, t) for r, t in case_type.relationships.iteritems()]
+        opened_by = [_get_translated_form_name(self.app, form_id, language)
+                     for form_id in case_type.opened_by.keys()]
+        closed_by = [_get_translated_form_name(self.app, form_id, language)
+                     for form_id in case_type.closed_by.keys()]
+        for i in range(max(len(relationships), len(opened_by), len(closed_by))):
+            row = [case_type.name]
+            row.append(relationships[i] if i < len(relationships) else '')
```

Comment:
do you think it would be useful to include the relationship in every row, so that you can filter the document?

Reply:
How would that work for multiple relationships? I suppose that I could only display the parent type in the row(s) containing an open/close form that's relevant to that relationship...would that work for host relationships as well?

Reply:
Oh, I though this was a tuple of (relationship, opened, closed). I.e. in the case of multiple relationships, I'd want to see which form opened that particular relationship type. Not sure if that information is available anywhere. 
I.e. I'd expect:

|relationship | opened_by | closed_by|
|---------|-------|---------|
|[extension] host | host opening form | host closing form|
|[extension] host | host opening form 2 ||
|[child] parent | parent opening form| parent closing form|
|[child] parent | | parent closing form 2|

But maybe I'm overthinking it.

Reply:
No, that makes sense and I don't think would be much more effort. I'll take a stab at it.

Reply:
updated:

<img width="850" alt="screen shot 2017-09-20 at 10 35 03 am" src="https://user-images.githubusercontent.com/1486591/30649947-a11528c8-9def-11e7-9819-8e319dfbc63f.png">